<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagnosisDialog</class>
 <widget class="QWidget" name="DiagnosisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/Icons/dr-simi-farmacias-similares-logo-svg-vector.svg</normaloff>:/icons/Icons/dr-simi-farmacias-similares-logo-svg-vector.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #ffffff;
	color: #000000
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_01">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Código:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codeTextEdit">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_02">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Descripción:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="descriptionTextEdit">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_03">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Indicaciones:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="indicationsTextEdit">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>codeTextEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>indicationsTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
